yt_dlp/extractor/gamejolt.py

```diff
@@ -0,0 +1,540 @@
+        'playlist_mincount': 2,
+        'info_dict': {
+            'id': '655124',
+            'title': 'Friday Night Funkin\': Friday 4 Fun',
+            'description': 'md5:576a7dd87912a2dcf33c50d2bd3966d3'
+        },
+        'params': {
+            'ignore_no_formats_error': True,
+        },
+        'expected_warnings': ['skipping format', 'No video formats found', 'Requested format is not available'],
+    }]
+
+    def _real_extract(self, url):
+        game_id = self._match_id(url)
+        game_data = self._call_api(
+            'web/discover/games/%s' % game_id, game_id, note='Downloading game info', errnote='Unable to download game info')['game']
+        description = self._parse_content_as_text(
+            self._parse_json(game_data.get('description_content', '{}'), game_id, fatal=False) or {})
+        return self.playlist_result(
+            self._entries('web/posts/fetch/game/%s' % game_id, game_id, 'Downloading game posts', 'Unable to download game posts'),
+            game_id, game_data.get('title'), description)
+
+
+class GameJoltGameSoundtrackIE(GameJoltBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?gamejolt\.com/get/soundtrack(?:\?|\#!?)(?:.*?[&;])??game=(?P<id>(?:\d+)+)'
+    _TESTS = [{
+        'url': 'https://gamejolt.com/get/soundtrack?foo=bar&game=657899',
+        'info_dict': {
+            'id': '657899',
+            'title': 'Friday Night Funkin\': Vs Oswald',
+            'n_entries': None,
+        },
+
+        'playlist': [{
+            'info_dict': {
+                'id': '184434',
+                'ext': 'mp3',
+                'title': 'Gettin\' Lucky (Menu Music)',
+                'url': r're:^https://.+vs-oswald-menu-music\.mp3$',
+                'release_timestamp': 1635190816,
+                'release_date': '20211025',
+                'n_entries': 3,
+            }
+        }, {
+            'info_dict': {
+                'id': '184435',
+                'ext': 'mp3',
+                'title': 'Rabbit\'s Luck (Extended Version)',
+                'url': r're:^https://.+rabbit-s-luck--full-version-\.mp3$',
+                'release_timestamp': 1635190841,
+                'release_date': '20211025',
+                'n_entries': 3,
+            }
+        }, {
+            'info_dict': {
+                'id': '185228',
+                'ext': 'mp3',
+                'title': 'Last Straw',
+                'url': r're:^https://.+last-straw\.mp3$',
+                'release_timestamp': 1635881104,
+                'release_date': '20211102',
+                'n_entries': 3,
+            }
+        }]
+    }]
+
+    def _real_extract(self, url):
+        game_id = self._match_id(url)
+        game_overview = self._call_api(
+            'web/discover/games/overview/%s' % game_id, game_id, note='Downloading soundtrack info', errnote='Unable to download soundtrack info')
+        return self.playlist_result(itertools.chain({
```

Comment:
am I missing something, or is the `itertools.chain` unnecessary?

Reply:
A bit of force of habit 😅